src-electron/main-process/modules/wallet-rpc.js

```diff
@@ -1243,26 +1298,24 @@ export class WalletRPC {
           });
           return;
         }
-
+        // update state to show a confirm popup
         this.sendGateway("set_tx_status", {
-          code: 0,
-          i18n: "notification.positive.sendSuccess",
-          sending: false
-        });
-
-        if (data.result) {
-          const hash_list = data.result.tx_hash_list || [];
-          // Save notes
-          if (note && note !== "") {
-            hash_list.forEach(txid => this.saveTxNotes(txid, note));
+          code: 1,
+          // TODO: translate
+          message: "Awaiting confirmation",
```

Comment:
Add a translation for this please. if you set the `i18n` key it'll pull the string directly.